newsApi.js

```diff
@@ -0,0 +1,24 @@
+const apiKey = require('./apiKey')
+
```

Comment:
NIce Api Key insertion! I need to get to tackling this, it seemed to just work with 'test' for me, which I hope it still does when you're looking at it too!